microservices/searchservice/config/config.yaml

```diff
@@ -2,14 +2,15 @@ api_prefix: $API_PREFIX
 base_url_search: $BASE_URL_SEARCH
 base_url_ingest: $BASE_URL_INGEST
 base_url_embedding: $BASE_URL_EMBEDDING
+deployment_env: $DEPLOYMENT_ENVIRONMENT
```

Comment:
should we maybe add this to every microservice config to keep it consistent?

Reply:
can do!